test/api/scenes.ts

```diff
@@ -227,3 +227,11 @@ test('SceneItem.addFile()', async t => {
   `),
   );
 });
+
+test('Try to make a not existing scene active', async t => {
+  const client = await getClient();
+  const scenesService = client.getResource<ScenesService>('ScenesService');
+  await sleep(30000);
```

Comment:
Yikes 30 seconds?  I hate to see a single test taking this long.  Is this valuable enough for that?

Reply:
It seems like my last `git push` that removes that failed. Should be fixed